signatory-ledger-tm/Cargo.toml

```diff
@@ -26,3 +26,11 @@ path = ".."
 
 [dev-dependencies]
 criterion = "0.2"
+
+[features]
+default = []
+#mockhsm = ["ledger-tm/mockhsm"]
```

Comment:
Do you want to keep this commented out? Maybe delete this line and the empty `default`?

Reply:
done